src/player/mod.rs

```diff
@@ -6,8 +6,8 @@ use heron::prelude::*;
 use leafwing_input_manager::prelude::*;
 use strum::EnumIter;
 
-static PLAYER_SIZE: isize = 1;
-static PLAYER_COLLISION_SIZE: f32 = PLAYER_SIZE as f32 / 2.;
+static PLAYER_SIZE: f32 = 1.0;
```

Comment:
`isize` stands for "integer of size of a pointer" btw. It seems reasonable to use a float here since it's converted into a float at all usage sites.